www/_data/toc/en_8-x-src.yml

```diff
@@ -0,0 +1,76 @@
+-
+    name: Introduction
+    children:
+        - url: guide/overview/index.html
+-
+    name: Create apps
+    children:
+        - url: guide/cli/index.html
+        - url: guide/cli/template.html
+        - url: guide/support/index.html
+        -
+            name: Develop for platforms
+            children:
+                - url: guide/platforms/android/index.html
+                - url: guide/platforms/blackberry10/home.html
+                - url: guide/platforms/ios/index.html
+                - url: guide/platforms/osx/index.html
+                - url: guide/platforms/ubuntu/index.html
+                - url: guide/platforms/win8/index.html
```

Comment:
can we change this URL to `windows` instead of `win8` somehow?

Reply:
@janpio we can, but make sure to redirect in `redirects.yml`